oam/install_scripts/post-install

```diff
@@ -206,24 +206,21 @@ fi
 #setup MariaDB Columnstore system logging
 if [ $user = "root" ]; then
 	$installdir/bin/syslogSetup.sh install > /tmp/syslog_install.log 2>&1
-	chmod 777 -R /dev/shm
 	rm -f /etc/default/columnstore
 else
-	$SUDO chmod 777 /tmp
 	$SUDO rm -fr /tmp/* > /dev/null 2>&1
 	$SUDO $installdir/bin/syslogSetup.sh --installdir=$installdir install > /tmp/syslog_install.log 2>&1
 	$SUDO chown $user:$user $installdir/etc/Columnstore.xml
 	$SUDO chmod -R 777 /dev/shm
 	$SUDO mkdir /var/lock/subsys > /dev/null 2>&1
 	$SUDO chmod 777 /var/lock/subsys > /dev/null 2>&1
 	$SUDO rm -f /var/lock/subsys/mysql-Columnstore
-	$SUDO chmod 777 /etc/fstab	
+	$SUDO chmod 644 /etc/fstab	
```

Comment:
If ColumnStore needs to edit /etc/fstab during execution then 666 would be more appropriate for non-root.